config/ompi_check_libfca.m4

```diff
@@ -25,41 +25,37 @@ AC_DEFUN([OMPI_CHECK_FCA],[
     OPAL_CHECK_WITHDIR([fca], [$with_fca], [lib/libfca.so])
```

Comment:
i am doing that right now
i also fixed the same inaccurate string in hcoll, mxm (and knem too)